### PR TITLE
Update analytics SDKs

### PR DESCRIPTION
**Pull request by logason**
Update Segment and Fullstory SDKs to latest versions – also bumped the Avo CLI version

### CODE REVIEW

File: Avo.ts [hunk 1]
```diff
@@ -1029,13 +1034,7 @@ let SegmentInspectorForDocs = {
   /* Segment does not have revenue api, @see https://segment.com/docs/spec/track/#properties */
   logEvent: function logEvent(eventName: any, eventProperties: any, integrations: any) {
     if (typeof (window as any) === 'undefined') { return; }
-    try {
-      (this as any).getInstance().track(eventName, eventProperties, Object.assign({}, {integrations: integrations}));
-    } catch(e) {
-      if(console && console.error) { 
-        console.error('Segment track error', e);
-      }
-    }
```
Comment (logason) [inline Avo.ts:1038]: Maybe we want to keep this try-catch until we've verified it ain't a problem any more?

**Review by hugihlynsson — COMMENTED** [review]
I'm still getting the error on the Vercel build 🤔
Screenshot 2021-07-07 at 23 31 38

**Comment by logason** [comment]
> I'm still getting the error on the Vercel build 🤔

We're yet to bump to 2.0 in Segment's dashboard right? There's still hope 🙏

**Comment by hugihlynsson** [comment]
> > I'm still getting the error on the Vercel build 🤔
> 
> We're yet to bump to 2.0 in Segment's dashboard right? There's still hope 🙏

Ahh yes!

**Review by hugihlynsson — APPROVED** [review]

**Pull request merged**